packages/trader/src/AppV2/Components/TradeParameters/Stake/stake.tsx

```diff
@@ -52,7 +52,7 @@ const Stake = observer(({ is_minimized }: TStakeProps) => {
         message: message_2 = '',
         payout: payout_2 = 0,
     } = proposal_info[contract_types[1]] ?? {};
-    const is_loading_proposal = !id_1 || (!!contract_types[1] && !id_2);
+    const is_loading_proposal = !has_error_1 && !has_error_2 && (!id_1 || (!!contract_types[1] && !id_2));
```

Comment:
this fixes an issue with being unable to click 'Save' when only one of two (e.g. Rise or Fall) contracts has max payout error.